docs/man_pages/lib-management/kendoui-install.md

```diff
@@ -0,0 +1,51 @@
+kendoui install
+==========
+
+Usage | Synopsis
+------|-------
+List all versions | `$ appbuilder kendoui install [--verified] [--latest]`
```

Comment:
How come we don't have the option to pass a package ID/index to $ appbuilder kendo ui install (for non-interactive consoles)